Calculate each section when generating string (#63)
nvim-lua/plenary.nvim: README.md

A Lua module that implements a bunch of the things from `pathlib` from Python, so that paths are easy to work with.

### plenary.scandir

`plenery.scandir` is fast recursive file operations. It is similar to unix `find` or `fd` in that it can do recursive scan over a given directory, or a set of directories. 

> It offers a wide range of opts for limiting the depth, show hidden and more. `plenary.scan_dir` can be ran asynchronously `on_insert(file, typ)` and `on_exit(files)` as well as synchronously

Author: Oh f. @tami5 `on_insert` and `on_exit` are available async but `on_insert` and only `on_insert` is also available for sync.

Reviewer: :D

Author: I think its not clear in the new version. Is it?

Reviewer: I think omit the function names, they can read tests and scan_dir file for more usage

A Lua module that implements a bunch of the things from `pathlib` from Python, so that paths are easy to work with.

### plenary.scandir

> A Lua module for fast recursive file operations. Offers a recursive scan over a directory, similar to unix `find` or `fd`.
> It should be faster than running unix `find` running in a `plenary.job` and as fast as `fd` in a `plenary.job`.

Reviewer: ```suggestion
`plenery.scandir` is fast recursive file operations. It is similar to unix `find` or `fd` in that it can do recursive scan over a given directory, or a set of directories. 
```

Author: I like the part where we are faster than find and as fast as fd but thanks :)

> It offers a wide range of opts for limiting the depth, show hidden and more. We also have a async version with callbacks for `on_insert(file, typ)` (also available for sync version) and `on_exit(files)`.

Reviewer: ```suggestion
It offers a wide range of opts for limiting the depth, show hidden and more. `plenary.scan_dir` can be ran asynchronously `on_insert(file, typ)` and `on_exit(files)` as well as synchronously
```
